lib/importer/src/index.js

```diff
@@ -207,9 +208,73 @@ exports.Initialise = (config, router, prototypeKit) => {
       headers: headers,
       totalCount: mapResults.totalCount,
       extraRecordCount: mapResults.extraRecordCount,
+      errorCount: mapResults.errorCount,
+      warningCount: mapResults.warningCount,
     };
   });
 
+  //--------------------------------------------------------------------
+  // Helper functions that can be used on the review page to show
+  // information about the data that has been mapped.
+  //--------------------------------------------------------------------
+  prototypeKit.views.addFunction(
+    "data_sum",
+    (data, column) => {
+      const session = data[IMPORTER_SESSION_KEY];
+      const mapResults = sheets_lib.MapData(session);
+      const headers = session.fields;
+
+      const idx = headers.findIndex((x) => x == column)
+      if (idx == -1) {
+        return "No data found"
+      }
+
+      let numbers = mapResults.resultRecords
+        .filter((x) => x !== undefined && x[idx] !== undefined)
+        .map((x) => { console.log(x); return x[idx].match(/\d+/g).join("") })
```

Comment:
Also, there is a rogue debug print in here that's printing all my lovely personal data to the console :)

Reply:
Removed it. But unfortunately left the {} in the lambda and then spent 20 minutes working out why the result was an array full of undefineds :|